src/ProgressOnderwijsUtils/Pocos/PocoUtils.cs

```diff
@@ -91,4 +91,14 @@ public static IPocoProperties<IPocoProperty> GetProperties(Type t)
 
     static readonly MethodInfo genericMethodInfo_GetProperties = Utils.F(GetProperties<object>).Method.GetGenericMethodDefinition();
     static readonly Func<Type, IPocoProperties<IPocoProperty>> memoizedGetProperties = Utils.F((Type t) => (IPocoProperties<IPocoProperty>)genericMethodInfo_GetProperties.MakeGenericMethod(t).Invoke(null, null).AssertNotNull()).ThreadSafeMemoize();
+
+    [Pure]
+    public static string PropertiesDiffLog<TPoco>(TPoco a, TPoco b)
+        => GetProperties<TPoco>()
```

Comment:
Dit is zo'n plekje waar trouwens denk ik de LINQ syntax beter leest (o.a. omdat je dan de AssertNotNull kan vermijden), dus grofweg:
```C#
from prop in GetProperties<TPoco>()
let getter = p.Getter
where getter is not null
let aVal = getter(a)
let bVal = getter(b)
where !Equals(aVal, bVal)
select $"{prop.Name}='{aVal}'»»'{bVal}'"
```